src/components/ArticleList.jsx

```diff
@@ -0,0 +1,43 @@
+import {useEffect, useState} from "react"
+import {getArticles} from "../api"
+import ArticleCard from "./ArticleCard"
+
+function ArticleList(chosenTopic){
+    const [articles, setArticles] = useState([])
+    const [err, setErr] = useState(null)
+
+    useEffect(()=>{
+        getArticles(chosenTopic)
```

Comment:
For this ticket, there is no requirement to sort by topic, so there shouldn't be any arguments being passed in here.